android/src/oldarch/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -0,0 +1,453 @@
+package com.applovin.reactnative;
+
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.ReadableArray;
+import com.facebook.react.bridge.ReadableMap;
+
+import java.util.Map;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+public class AppLovinMAXModule
```

Comment:
Is there a way to force pubs to new arch? So we don't maintain this and the one for newarch

Reply:
No, there is no way.   `newArchEnabled` in `android/gradle.properties` in the pubs' app controlls the arch.   On iOS, the build environment `RCT_NEW_ARCH_ENABLED` controlls it when running 'pod install'.